google/ggrc-core
# GGRC-6182, GGRC-6314, GGRC-6136, GGRC-6151, GGRC-6166, GGRC-6181, GGRC-6246 Holistic solution for Assessment - IssueTracker sync (Phase one)

# Dependencies
- [x]  ~~On hold until we will merge 1.38 into release.~~
# Issue description
Implement only part of the solution, update assessment_integration.py module, add one class for Assessment integration and include all necessary logic inside this class (builders and handler). It isn't a complete solution, but it's a help to improve current implementation for Assessment and Audit integration, all necessary logic will be included into one class and used in places where it's necessary

Related ticket, include BA requirements:
GGRC-6166

# Steps to test the changes
Re-check all actions, connected with Assessment integration with Issue Tracker
Run integration/unit tests


# Sanity checklist















# PR Review checklist
